assets/css/components/_button-detail.scss

```diff
@@ -0,0 +1,5 @@
+.button-detail {
```

Comment:
Will this class be reused anywhere? If not, I think it would be simpler if we directly apply classes.

```elixir
<span><%= link "View", class: "btn btn-primary btn-sm", to: Routes.keyword_path(@conn, :show, keyword) %></span>
```

Reply:
hi @hanam1ni !
I am also not sure what is the best approach to use here, please take a look at my previous PR: https://github.com/byhbt/kwtool/pull/18#discussion_r603039718

As Olivier suggested that we should not use the utility classes from CSS framework directly in our HTML code. 
However in this case, which one I should use?

Reply:
The case you mentioned is for utility classes. But this one is a class component and its variant class that can be used together in an element class.

Reply:
Currently, the class `.button-detail` i just use this class for the `view` detail button in the list of keywords. It's `sm` size button.

![Screen Shot 2021-05-04 at 2 02 47 PM](https://user-images.githubusercontent.com/948856/116970014-6a8a4e00-ace1-11eb-94f2-87248fdbe7fb.png)